Common/include/config_structure.hpp

```diff
@@ -1878,6 +1947,66 @@ class CConfig {
    */
   su2double GetThermalDiffusivity(void);
   
+  /*!
+   * \brief Get the thermal expansion coefficient.
+   * \return Value of the thermal expansion coefficient.
+   */
+  su2double GetThermal_Expansion_Coeff(void);
+
+  /*!
+   * \brief Get the non-dim. thermal expansion coefficient.
+   * \return Value of the non-dim. thermal expansion coefficient.
+   */
+  su2double GetThermal_Expansion_CoeffND(void);
+
+  /*!
+   * \brief Set the thermal expansion coefficient.
+   * \param[in] val_thermal_expansion - thermal expansion coefficient
+   */
+  void SetThermal_Expansion_Coeff(su2double val_thermal_expansion);
+
+  /*!
+   * \brief Set the non-dim. thermal expansion coefficient.
+   * \param[in] val_thermal_expansion - non-dim. thermal expansion coefficient
+   */
+  void SetThermal_Expansion_CoeffND(su2double val_thermal_expansionnd);
+
+  /*!
+   * \brief Get the value of the reference density for custom incompressible non-dimensionalization.
+   * \return Reference density for custom incompressible non-dimensionalization.
+   */
+  su2double GetInc_Density_Ref(void);
+
+  /*!
+   * \brief Get the value of the reference velocity for custom incompressible non-dimensionalization.
+   * \return Reference velocity for custom incompressible non-dimensionalization.
+   */
+  su2double GetInc_Velocity_Ref(void);
+
+  /*!
+   * \brief Get the value of the reference temperature for custom incompressible non-dimensionalization.
+   * \return Reference temperature for custom incompressible non-dimensionalization.
+   */
+  su2double GetInc_Temperature_Ref(void);
```

Comment:
Does we need to use the Inc_ prefix? If we need to use it... are we using it for everything? Pressure, etc...

Reply:
This is only for the reference values in the non-dim scheme. For the incompressible solver, I have allowed the user to give directly the reference density, velocity, and temp (if not isothermal), like we originally did for the compressible code. The incompressible solver now uses almost the same non-dim scheme as the compressible code, other than the definition of the reference pressure (rho_ref*V_ref*V_ref).